po/newsboat.pot

```diff
@@ -476,8 +476,7 @@ msgstr ""
 
 #: src/controller.cpp:564
 msgid ""
-"Unreachable feeds found, consider setting `cleanup-on-quit yes` or run "
-"`newsboat --cleanup`"
+"%d unreachable feeds found. See `cleanup-on-quit` in newsboat(1) for details."
```

Comment:
Please don't touch this file at all, it will get updated automatically before the release.

Reply:
I reverted this change.